des.css

```diff
@@ -0,0 +1,180 @@
+/* Desktop version set up */
+.open {
+  display: none;
+}
+
+/* Top media bar set up */
+.top {
+  display: block;
+  position: fixed;
+  width: 100%;
+}

```

Comment:
- [ ] Good job implementing all the required changes, however this is a mobile first approach towards web development and as such you have to use media queries to change from mobile to desktop and vice-versa. I am unable to find any media query where your app is changing from mobile to desktop, kindly implement this.

example
![image](https://user-images.githubusercontent.com/81932478/205454343-0820d2b1-7dad-47b1-985c-d690146e7969.png)